main.go

```diff
@@ -62,6 +64,8 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:             scheme,
+		Namespace:          namespace,
+		NewCache:           cache.MultiNamespacedCacheBuilder([]string{namespace}),
```

Comment:
I took it from https://book.kubebuilder.io/cronjob-tutorial/empty-main.html#every-journey-needs-a-start-every-program-a-main and reading https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder and https://github.com/argoproj-labs/applicationset/blob/master/main.go#L123

It seems to me that makes sense to restrict the cache since we're watching a single namespace

Reply:
After reading https://github.com/kubernetes-sigs/controller-runtime/blob/v0.9.2/pkg/manager/manager.go#L194 it seems we just need to specify `Namespace`